lib/cmd/common/configuration.js

```diff
@@ -27,22 +27,22 @@ function configuration(argv, cb) {
 
   // defaults.. 'list all' operation if 1 args passed, 'list <dest>' if 2..
   if (args.length === 1) {
-    return list(fhc.appId(args[0]), 'all', cb);
+    return list(args[0], 'all', cb);
```

Comment:
1) Looks like signature of the command changed here. 
2) `args[0]` is not the best way to address yargs arguments. Let's try to use names or map positional arguments into names.